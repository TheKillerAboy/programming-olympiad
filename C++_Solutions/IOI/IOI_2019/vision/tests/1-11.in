c2675211-ade0-44b0-8c15-741dd835f3d2
3 2 2
-1
