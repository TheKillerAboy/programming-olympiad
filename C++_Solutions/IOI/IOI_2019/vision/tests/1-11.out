b17553fd-ba5a-4140-836c-491f938c515b
OK
64
1 1 0
2 1 0
1 2 1 2
2 2 1 2
1 2 3 4
2 2 3 4
1 1 5
2 1 5
1 1 1
2 1 1
1 2 0 3
2 2 0 3
1 2 2 5
2 2 2 5
1 1 4
2 1 4
2 2 15 17
1 2 14 16
3 22
0 2 23 24
2 2 19 17
1 2 18 16
3 26
0 2 27 28
2 2 19 21
1 2 18 20
3 30
0 2 31 32
1 3 25 29 33
2 2 7 9
1 2 6 8
3 35
0 2 36 37
2 2 11 9
1 2 10 8
3 39
0 2 40 41
2 2 11 13
1 2 10 12
3 43
0 2 44 45
1 3 38 42 46
0 2 47 34
2 3 15 17 19
1 3 14 16 18
3 49
0 2 50 51
2 3 21 17 19
1 3 20 16 18
3 53
0 2 54 55
1 2 52 56
2 3 7 9 11
1 3 6 8 10
3 58
0 2 59 60
2 3 13 9 11
1 3 12 8 10
3 62
0 2 63 64
1 2 61 65
0 2 66 57
3 48
0 2 67 68
